### ISSUE TITLE
Cannot read properties of undefined (reading 'ar') w/ Elixir 1.4.2/OTP 25

### ISSUE THREAD

**Issue by fhunleth** [scout119/beamdasm#303]
[Elixir-1.14.2-OTP-25-Enum.zip](https://github.com/scout119/beamdasm/files/10327836/Elixir-1.14.2-OTP-25-Enum.zip) contains an example beam file that shows the error. This is from the [Elixir 1.14.2 pre-built release](https://github.com/elixir-lang/elixir/releases/download/v1.14.2/elixir-otp-25.zip). 

I didn't see any other logs to share, but please let me know if there's anything else helpful that I could provide. 

Thank you for the very convenient disassembler!

**Comment by ryanwinchester** [comment]
Related to #6?

**Comment by mhanberg** [comment]
Same

**Comment by kotuk** [comment]
Same with OTP 25 and 26

**Comment by kumaraguru1735** [comment]
yes same issue for me, if anyone has a solution comment here out...
Thank you
